tools/run-converter.sh

```diff
@@ -21,3 +21,11 @@ dotnet run --project $CONVERTER_PROJECT -- \
   $SPEC_DIRECTORY/*.md $TEMPLATE \
   -o $OUTPUT_DIRECTORY/standard.docx
 
+if [ "$?" -eq "0" ]
```

Comment:
Possibly just check whether GITHUB_OUTPUT is non-empty?

Reply:
I was following the guidance here: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#environment-files

In all there examples, and the guidance uses `>>` and doesn't check for a non-empty environment variable. 

I agree to keep watching this, but I'll merge as is for now.

Reply:
> In all there examples, and the guidance uses >> and doesn't check for a non-empty environment variable.

I suspect that was because those examples are for code which is only expected to run as part of a GitHub action.

I run this shell script locally when testing the converter - what do we expect it to do in that case? (I have no idea what `>>` does if you don't tell it what to write to...)

Reply:
Gotcha.  

I ran it locally, and that command fails with "ambiguous redirect".

I'll make a fresh PR that only sets those if the environment is set. (That's in multiple actions).